packages/client/components/TeamPrompt/Recurrence/RecurrenceTimePicker.tsx

```diff
@@ -0,0 +1,30 @@
+import dayjs from 'dayjs'
+import ms from 'ms'
+import React from 'react'
+import {MenuProps} from '../../../hooks/useMenu'
+import Menu from '../../Menu'
+import MenuItem from '../../MenuItem'
+
+interface Props {
+  menuProps: MenuProps
+  onClick: (n: Date) => void
+}
+const options = [...Array(96).keys()].map((n) => n * ms('15m'))
+export const RecurrenceTimePicker = (props: Props) => {
+  const {menuProps, onClick} = props
+  const startOfToday = new Date().setHours(0, 0, 0, 0)
+  return (
+    <Menu {...menuProps} ariaLabel={'6:00 AM'}>
```

Comment:
-1 can we add a `defaultActiveIdx` on & scroll to the current value? For example, if the value is currently 6AM, when i open the menu that item should be highlighted & in the center so I can quickly pick 5:30 or 6:30

Reply:
added `defaultActiveIdx`. seems like it should automatically scroll to the element [using this code.](https://github.com/ParabolInc/parabol/blob/1f4ae32fddb5c74e7cae8d2da6516933787afd10/packages/client/components/MenuItem.tsx#L60) but it doesn't seem to work. I played with it for a while but I can't make it work. would you mind if I handle this as a separate issue?